lib/ansible/modules/network/fortios/fortios_wireless_controller_wtp_profile.py

```diff
@@ -0,0 +1,1820 @@
+                                           "AO", "AR", "AM",
+                                           "AU", "AT", "AZ",
+                                           "BH", "BD", "BB",
+                                           "BY", "BE", "BZ",
+                                           "BO", "BA", "BR",
+                                           "BN", "BG", "KH",
+                                           "CL", "CN", "CO",
+                                           "CR", "HR", "CY",
+                                           "CZ", "DK", "DO",
+                                           "EC", "EG", "SV",
+                                           "EE", "FI", "FR",
+                                           "GE", "DE", "GR",
+                                           "GL", "GD", "GU",
+                                           "GT", "HT", "HN",
+                                           "HK", "HU", "IS",
+                                           "IN", "ID", "IR",
+                                           "IE", "IL", "IT",
+                                           "JM", "JO", "KZ",
+                                           "KE", "KP", "KR",
+                                           "KW", "LV", "LB",
+                                           "LI", "LT", "LU",
+                                           "MO", "MK", "MY",
+                                           "MT", "MX", "MC",
+                                           "MA", "MZ", "MM",
+                                           "NP", "NL", "AN",
+                                           "AW", "NZ", "NO",
+                                           "OM", "PK", "PA",
+                                           "PG", "PY", "PE",
+                                           "PH", "PL", "PT",
+                                           "PR", "QA", "RO",
+                                           "RU", "RW", "SA",
+                                           "RS", "ME", "SG",
+                                           "SK", "SI", "ZA",
+                                           "ES", "LK", "SE",
+                                           "SD", "CH", "SY",
+                                           "TW", "TZ", "TH",
+                                           "TT", "TN", "TR",
+                                           "AE", "UA", "GB",
+                                           "US", "PS", "UY",
+                                           "UZ", "VE", "VN",
+                                           "YE", "ZB", "ZW",
+                                           "JP", "CA"]},
+                "ble-profile": {"required": False, "type": "str"},
+                "comment": {"required": False, "type": "str"},
+                "control-message-offload": {"required": False, "type": "str",
+                                            "choices": ["ebp-frame", "aeroscout-tag", "ap-list",
+                                            
```

Comment:
Please refrain from using `global`. Instead, pass it to the methods that use this.

Reply:
Right. Actually some methods already include the variable as an argument. I'll add a fix for this.